src/main/java/net/datafaker/Computer.java

```diff
@@ -0,0 +1,42 @@
+package net.datafaker;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Computer {
+
+    private final Faker faker;
+
+    protected Computer(Faker faker) {
+        this.faker = faker;
+    }
+
+    public String type() {
+        return faker.resolve("computer.type");
+    }
+
+    public String platform() {
+        return faker.resolve("computer.platform");
+    }
+
+    public String operatingSystem() {
+        List<String> operatingSystems = new ArrayList<>();
+        operatingSystems.add("linux");
+        operatingSystems.add("macos");
+        operatingSystems.add("windows");
+        return faker.resolve("computer.os." + operatingSystems.get(faker.random().nextInt(0, 2)));
```

Comment:
@bodiam Is there a better way to implement this?

Reply:
```java
return faker.resolve("computer.os." + faker.options().option("linux", "macos", "windows"));
```

Reply:
You could also use an Enum here